reference/commands.rst

```diff
@@ -45,6 +46,7 @@ and these :ref:`custom command examples <examples_extensions_custom_commands>`
 - :doc:`conan remote <commands/remote>`: Add, remove, login/logout and manage remote server
 - :doc:`conan search <commands/search>`: Search packages matching a name
 - :doc:`conan version <commands/version>`: Give information about the Conan client version
+- :doc:`conan workspace (incubating) <commands/workspace>`: Manage Conan workspaces (group of packages in editable mode)
```

Comment:
I think that better than putting this: `(group of packages in editable mode)` just provide a link for workspaces section in `Manage Conan workspaces`?

Reply:
I mean omitting the `(group of packages in editable mode)` or replace it with a link in the work `workspaces` to the place where you explain that a workspace is a group of packages in editable mode.
```suggestion
- :doc:`conan workspace (incubating) <commands/workspace>`: Manage Conan workspaces
```